.github/workflows/on-pr-and-trunk_one-app-unit-and-lint-tests.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x]
```

should actually remove 14.x like the title says
```suggestion
        node-version: [16.x]
```